dimmer.el

```diff
@@ -12,15 +12,15 @@
 ;;; Commentary:
 ;; 
 ;; This module provides a subtle visual indication which window is
-;; currently active by dimming the faces on the others. It does this
+;; currently active by dimming the faces on the others.  It does this
 ;; nondestructively, and computes the dimmed faces dynamically such
```

Comment:
This is just funny.  Elsewhere in my life I get called out for being a habitual two-spacer, but monospaced code is a place where it can still be argued...

Reply:
`M-x checkdoc` insists on it.

Reply:
I know, it's just funny how I have to keep remembering what situation I'm in. I added a Travis CI test for checkdoc by the way.